### ISSUE TITLE
Moving sidebar from right to left by default

### ISSUE THREAD

**Issue by ucalegonte** [bootscore/bootscore#188]
I apologize if the question may be silly, but I am new to wordpress and I don't know if there is already a tip for this.
What is the best way to move the sidebar from right to left as default without having to manually duplicate and correct all the template files in the child theme?
I have seen that for each page or post it is possible to manually change the template, but it is easy to forget to do it every time.
And then, another question, if the site is set to show the Blog Posts Index as the front page, how can I change its position from right to left on this page as well?
Thank you

**Comment by crftwrk** [comment]
Please do all issues regarding to child-themes here in main repo. It's better for us to keep an eye on them. However, your question is not silly and simple to solve. Check here https://bootscore.me/documentation/page-post-templates/.

1. Go in main theme's root folder and copy `page-sidebar-left.php` and `single-sidebar-left.php` to your child-theme's root folder.
2. Rename files to `page.php` and `single.php`. Now sidebar-left templates are the default ones.
3. Do the same with `category-*.php`, `author-*.php` and `archive-*.php` if you want to use sidebar on the left side in archives as well.

For the `index.php` it's nearly the same. 

1. Copy this file from main-theme root folder to your child's root. 
2. Open this file in an editor.
3. In line `157` you will find this `<?php get_sidebar(); ?>`
4. Move snippet above the main column to line `103`

```html
<div class="row">
  <!-- Paste sidebar here -->
  <?php get_sidebar(); ?>
  <div class="col col-md-8 col-xxl-9">
```

Does this help?

**Comment by crftwrk** [comment]
👍 


For the comment form, yes you‘re right. This should be a pluggable function. Do you want to create a pull request for this?

**Comment by ucalegonte** [comment]
Well, since this is the only way to modify the CSS inside, I think it's a good idea to make it pluggable, so yes, I'll pull a request for this.

Thank you

**Comment by crftwrk** [comment]
Close this issue now as resolved.

Note that we've moved in 5.2.0.0 update page and single templates to subfolders https://bootscore.me/blog/bootstrap/bootscore-5-2-0-0/. If you want to update, your commands should be this:

`>sudo ln -s ../bootscore-main/page-templates/page-sidebar-left.php page.php`
`>sudo ln -s ../bootscore-main/single-templates/single-sidebar-left.php single.php`

We also removed all `archive-*.php`, `category-*php` and `author-*php` out of the theme and pushed them to an own repo https://github.com/bootscore/bs-loop-templates. Theme itself uses only the default `archive.php` now. If you want to use different templates for them, you must copy them to your child.

For the comment section, we're happy if you just fork the theme, wrap function in the `if (!function_exists...` and create a pull request with your changes. We will accept and merge it. 

Thank you
